seneca.js

```diff
@@ -891,9 +891,7 @@ function make_seneca (initial_options) {
     pin = _.isArray(pin) ? pin : [pin]
     _.each(pin, function (p) {
       _.each( pinthis.findpins(p), function (actpattern) {
-        pinthis.add(actpattern, meta, function (args, done) {
-          wrapper.call(this, args, done)
-        })
+        pinthis.add(actpattern, meta, wrapper)
       })
```

Comment:
Actually what the outer `function` does is to add another scope of `this` which we are getting rid of by the `call` so the `wrapper` would be called with the same this as specified by `add`. When you think about it this is exactly what happens when you pass any function to `add` and it is also true for user supplied `wrapper`.

In my sample code in #296 I'm using the `this` fom inside the wrappers and they still work correctly.